script.js

```diff
@@ -1,96 +1,229 @@
-//You can edit ALL of the code here
 function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+  var allShows = getAllShows()
+ var sortedShows =  allShows.sort((a, b) => a.name.localeCompare(b.name))
+  selectionOfShows(sortedShows)
+  listOfShowsProfiles(sortedShows)
+}
+
+//---------make a home button ---------//////
+let homeBtn = document.getElementById('homeBtn')
+homeBtn.addEventListener('clicked', setup)
+const rootElem = document.getElementById("root");
+const mainEle = document.createElement('div');
+mainEle.id = 'main'
+rootElem.appendChild(mainEle)
 
+/////////////////////////----Create list of shows for selection element----///////////////////////////////////
+let selectionBarForShow = document.querySelector('.select')
+const searchBar = document.forms['search-bar'].querySelector('input');
+function selectionOfShows(orderedShowList){
+  let option = document.createElement('option')
+  option.innerHTML = 'SELECT A SHOW'
+  selectionBarForShow.appendChild(option)
+  orderedShowList.forEach(show=> { 
+   const showOption = document.createElement('option')
+   showOption.innerText = show.name
+   showOption.value = show.id
+   selectionBarForShow.appendChild(showOption)
+  
+  })
 }
 
-function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  const mainEle = document.createElement('div');
-  mainEle.id = 'main'
-  rootElem.appendChild(mainEle)
-
-  //creating eposides and display them with their names and summary.
-  for (let i = 0 ; i < episodeList.length ; i++ ){
-    divEle = document.createElement('div');
-   divEle.id = 'episode'
-   mainEle.appendChild(divEle)
-   let titleEle = document.createElement('h1')
-   titleEle.id = 'title'
-   divEle.appendChild(titleEle)
-   titleEle.innerText = nameOfEpisodes(episodeList[i])
-   let EpisodeImage = document.createElement('img')
-      EpisodeImage.src = episodeList[i].image.medium;
-   divEle.appendChild(EpisodeImage);
-   let paraEle = document.createElement('div')
-   paraEle.id = 'para'
-   paraEle.innerHTML = episodeList[i].summary
-   divEle.appendChild(paraEle)
+   //////////-------list of Shows profiles--------------->
+function listOfShowsProfiles(orderedShows){
+  displayingNumOfEpisodes(orderedShows, orderedShows, 'SHOW/S')
 
+ ///////////////////////////////////////////////////////////
+ orderedShows.forEach(show=> {
+  let showProfile = document.createElement('div')
+  showProfile.id = 'show-profile'
+  let inShowProfile = document.createElement('div')
+  inShowProfile.id = 'inShowProfile'
+  let title = document.createElement('h1')
+  title.id = 'show-title'
+  title.innerText = show.name
+  let showImage = document.createElement('img')
+  if(show.image){
+    showImage.src = show.image.medium
+  } else {
+    showImage.src = 'https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRqEWgS0uxxEYJ0PsOb2OgwyWvC0Gjp8NUdPw&usqp=CAU'
```

Comment:
Great idea to get around situations where the data is unavailable, however, by using a link, you risk the link becoming unavailable at some point, so I would recommend saving the image to your project and navigating to it locally rather than using a URL